docs/tutorials/autogluon-cloud.md

```diff
@@ -1,108 +1,184 @@
 
-```{note}
-Make sure you review the trust relationship and IAM policy files, and make necessary changes according to your use case before applying them.
-```
+::::{tab-set}
+:::{tab-item} CloudFormation (AWS CLI)
+:sync: setup-cli
+1. Download and review the CloudFormation template from the [AutoGluon-Cloud repository](https://github.com/autogluon/autogluon-cloud/tree/add-cfn-templates/cloudformation)
+    ```bash
+    BACKEND="sagemaker"  # Supported options "sagemaker", "ray_aws"
+    wget https://raw.githubusercontent.com/autogluon/autogluon-cloud/refs/heads/add-cfn-templates/cloudformation/ag_cloud_$BACKEND.yaml
+    ```
+    ```{note}
+    Make sure you review the IAM policy defined in the CloudFormation template, and make necessary changes according to your use case before applying it.
+    ```
 
-We recommend you to create an IAM Role for your IAM User to delegate as IAM Role doesn't have permanent long-term credentials and is used to directly interact with AWS services. Here is how this can be done using the AWS CLI.
+2. Deploy the CloudFormation stack
+    ```bash
+    aws cloudformation create-stack \
+        --stack-name ag-cloud \  # use your preferred stack name
+        --template-body file://ag_cloud_$BACKEND.yaml \
+        --capabilities CAPABILITY_NAMED_IAM  # give permission to create IAM roles
+    ```
 
-```{note}
-Make sure to replace `AUTOGLUON-ROLE-NAME` with your desired role name, `AUTOGLUON-POLICY-NAME` with your desired policy name, and `222222222222` with your AWS account number.
-```
+3. Review the outputs produced by the stack
+    ```bash
+    aws cloudformation describe-stacks --stack-name ag-cloud --query "Stacks[0].Outputs"
+    ```
+    The output should contain the **name of the S3 bucket** and the **ARN of the IAM role** created for AutoGluon-Cloud.
+    ```json
+    [
+        {
+            "OutputKey": "BucketName",
+            "OutputValue": "ag-cloud-bucket-abcd1234",
+            "Description": "S3 bucket where AutoGluon-Cloud will save trained predictors"
+        },
+        {
+            "OutputKey": "RoleARN",
+            "OutputValue": "arn:aws:iam::222222222222:role/ag-cloud-execution-role",
+            "Description": "ARN of the created IAM role for AutoGluon-Cloud to run on SageMaker"
+        }
+    ]
+    ```
+
+:::
+:::{tab-item} CloudFormation (AWS Console)
+:sync: setup-cfn
+1. Download and review the CloudFormation template for the backend of your choice from the [AutoGluon-Cloud repository](https://github.com/autogluon/autogluon-cloud/tree/add-cfn-templates/cloudformation)
+    - Template for [SageMaker](https://raw.githubusercontent.com/autogluon/autogluon-cloud/refs/heads/add-cfn-templates/cloudformation/ag_cloud_sagemaker.yaml)
+    - Template for [Ray (AWS)](https://raw.githubusercontent.com/autogluon/autogluon-cloud/refs/heads/add-cfn-templates/cloudformation/ag_cloud_ray_aws.yaml)
```

Comment:
Links should be updated to `https://github.com/autogluon/autogluon-cloud/tree/master/cloudformation` after the PR is merged.